test/typescript-karma/Dashboards/Component/tests.js

```diff
@@ -478,7 +480,34 @@ test('HighchartsComponent resizing', function(assert) {
 
     const { width, height } = component.element.style
     assert.ok(true)
-})
+});
+
+test('Chart update in HighchartsComponent', function(assert) {
+    const parent = document.createElement('div');
+    parent.id = 'test';
+    parent.style.width = '500px';
+    document.getElementById('container').appendChild(parent);
+
+    const component = new HighchartsComponent({
+        parentElement: parent,
+        chartOptions: {
+            title: {
+                text: 'test'
+            },
+            series: [{
+                data: [1, 2, 3]
+            }]
+        }
+    }).render();
+
+    component.chart.update({
+        title: {
+            text: 'updated'
+        }
+    });
+
+    assert.deepEqual(component.options.chartOptions.title.text, 'updated');
```

Comment:
Do we need that `deepEqual` assertion? Isn't `srictEqual` enough?

Reply:
Good point!

Long time ago, I created tests like this one :D

Reply:
It depends. `deepEqual` is not strict. `strictEqual` is not deep.